tests/timeseries/_ev_charger_pool/test_ev_charger_pool_control_methods.py

```diff
@@ -0,0 +1,266 @@
+                "._ev_charger_manager._ev_charger_manager.ComponentPoolStatusTracker",
+                return_value=mock,
+            )
+        await mocks.component_status_sender.send(
+            ComponentPoolStatus(working=set(mocks.microgrid.evc_ids), uncertain=set())
+        )
+
+    async def _patch_data_pipeline(self, mocker: MockerFixture) -> None:
+        mocker.patch(
+            "frequenz.sdk.microgrid._data_pipeline._DATA_PIPELINE._ev_power_wrapper"
+            "._pd_wait_for_data_sec",
+            0.1,
+        )
+
+    async def _patch_power_distributing_actor(
+        self,
+        mocker: MockerFixture,
+    ) -> None:
+        dp = typing.cast(_DataPipeline, microgrid._data_pipeline._DATA_PIPELINE)
+        pda = typing.cast(
+            PowerDistributingActor, dp._ev_power_wrapper._power_distributing_actor
+        )
+        cm = typing.cast(
+            EVChargerManager,
+            pda._component_manager,
+        )
+        mocker.patch(
+            "frequenz.sdk.microgrid._data_pipeline._DATA_PIPELINE._ev_power_wrapper"
+            "._power_distributing_actor._component_manager._config",
+            EVDistributionConfig(
+                component_ids=cm._config.component_ids,
+                initial_current=cm._config.initial_current,
+                min_current=cm._config.min_current,
+                increase_power_interval=timedelta(seconds=0.12),
+            ),
+        )
+        mocker.patch(
+            "frequenz.sdk.microgrid._data_pipeline._DATA_PIPELINE._ev_power_wrapper"
+            "._power_distributing_actor._component_manager._voltage_cache.get",
+            return_value=Sample3Phase(
+                timestamp=datetime.now(tz=timezone.utc),
+                value_p1=Voltage.from_volts(220.0),
+                value_p2=Voltage.from_volts(220.0),
+                value_p3=Voltage.from_volts(220.0),
+            ),
+        )
+
+    async def _init_ev_chargers(self, mocks: _Mocks) -> None:
+        now = datetime.now(tz=timezone.utc)
+        for evc_id in mocks.microgrid.evc_ids:
+            mocks.streamer.start_streaming(
+                EvChargerDataWrapper(
+                    evc_id,
+                    now,
+                    cable_state=EVChargerCableState.EV_PLUGGED,
+                    component_state=EVChargerComponentState.READY,
+                    active_power=0.0,
+                    active_power_inclusion_lower_bound=0.0,
+                    active_power_inclusion_upper_bound=16.0 * 230.0 * 3,
+                    voltage_per_phase=(230.0, 230.0, 230.0),
+                ),
+                0.05,
+            )
+
+        for meter_id in mocks.microgrid.meter_ids:
+            mocks.streamer.start_streaming(
+                MeterDataWrapper(
+                    meter_id,
+                    now,
+                    voltage_per_phase=(230.0, 230.0, 230.0),
+                ),
+                0.05,
+            )
+
+        await asyncio.sleep(1)
```

Comment:
Is this really sleeping for one second? This will make tests really slow :sob:

Reply:
Probably not necessary, but I don't want to find out now.  Will deal with it when migrating to solipsism.